examples/go_extension/todo.yaml

```diff
@@ -0,0 +1,70 @@
+extensions:
+- id: go_extension
+  code_type: go
+  path: ""
+  url: file://go_extension.so
+- action: read
+  effect: allow
+  id: member_schema
+  principal: Member
+  resource:
+    path: /gohan/v0.1/schemas*
+- action: '*'
+  condition:
+  - is_owner
+  effect: allow
+  id: member_policy
+  principal: Member
+  resource:
+    path: /v0.1/member_resources*
+    properties:
+    - id
+    - name
+    - description
+    - tenant_id
+schemas:
+- description: Todo
+  id: todo
+  plural: todos
+  prefix: /v0.1
+  schema:
+    properties:
+      description:
+        description: Description
+        default: ""
+        permission:
+        - create
+        - update
+        title: Description
+        type: string
+        unique: false
+      id:
+        description: ID
+        permission:
+        - create
+        title: ID
+        type: string
+        unique: false
+      name:
+        description: Name
+        permission:
+        - create
+        - update
+        title: Name
+        type: string
+        unique: false
+      tenant_id:
+        description: Tenant ID
+        permission:
+        - create
+        title: Tenant
+        type: string
+        unique: false
+    propertiesOrder:
+    - id
+    - name
+    - description
+    - tenant_id
+    type: object
+  singular: todo
```

Comment:
yes, there are two possible ways of registering a handler:

* a general handler for standard or custom event (no schema)

> RegisterEventHandler(eventName string, handler func(context Context, environment IEnvironment) error, priority Priority)

* a schema handler for standard or custom event:

> 	RegisterSchemaEventHandler(schemaID string, eventName string, handler func(context Context, resource Resource, environment IEnvironment) error, priority Priority)